src/unix/linux/cpu.rs

```diff
@@ -105,7 +104,7 @@ impl CpusWrapper {
                     let mut parts = line.split(|x| *x == b' ').filter(|s| !s.is_empty());
                     if first {
                         self.global_cpu.inner.name =
-                            to_str!(parts.next().unwrap_or(&[])).to_owned();
+                            OsStr::from_bytes(parts.next().unwrap_or(&[])).to_owned();
```

Comment:
The previous `to_str!` also felt a little dubious here considering it just always assumed valid UTF-8, even though that's afaict not a guarantee that Linux gives here.